### ISSUE TITLE
Is loss function applied to fc8 layer, but not to sigmoid layer?

### ISSUE THREAD

**Issue by ghost** [ssudholt/phocnet#6]
Does the code  'n.loss = L.SigmoidCrossEntropyLoss(n.fc8, n.phocs)'  implies that cross entropy loss function was used with the output of last fully connected layer with linear activation?

**Comment by ssudholt** [comment]
In Caffe, the SigmoidCrossEntropyLoss layer combines both sigmoid activation and the binary cross entropy loss in one layer in order to provide numerical stability. At test/retrieval time, the sigmoid activation is added as a dedicated layer in order to get the output of fc8 in the correct range.

**Comment by ghost** [comment]
Thank you very much!
